src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    private static final int max_num = 100;
+    private final Random random = new Random();
+    private final ColorSupplier supplier = new ColorSupplier();
```

Comment:
Let's pass these objects to the constructor as I suggested in the message in the group chat)